concourse/pipelines/deploy.yml

```diff
@@ -818,19 +834,29 @@ jobs:
         passed:
         - run-terraform
         trigger: true
-      - get: release
-        resource: static
+      - get: static-image
         trigger: true
+    - task: upload-rails-assets
+      file: govuk-infrastructure/concourse/tasks/upload-rails-assets.yml
+      input_mapping:
+        app-image: static-image
+      params:
+        IMAGE_ASSETS_PATH: image/rootfs/app/public/assets/static
+        S3_BUCKET_PATH: s3://govuk-test-ecs-rails-assets/assets/static/
```

Comment:
I think this could happen in parallel with the task definition update (might be a small performance win).

Reply:
I can do that